Makefile

```diff
@@ -1,4 +1,4 @@
-NAME = bufferapp/manual-payments:0.2.0
+NAME = bufferapp/manual-payments:0.3.0
```

Comment:
:+1: 

We probably also want to bump the version number here: https://github.com/bufferapp/manual-payments/blob/master/cronjob.yaml#L14

Reply:
Good call!

Reply:
Updated the version number here: https://github.com/bufferapp/manual-payments/pull/8/commits/86fa1ded18b8b34c896490be35c1cee430fecd4e